src/styles/components/_blockquote.scss

```diff
@@ -1,28 +1,10 @@
-@import '../settings/type';
-@import '../tools/mixins';
-
 // =============================================================================
 // BLOCKQUOTE OVERVIEW
 //
 // A large callout quote with an optional author field.
 //
 // =============================================================================
 
-.w-blockquote {
-  @include w-article-text();
-  // DevSite override
-  background: transparent;
-  margin: 32px 0;
-  padding-left: 48px;
-}
-
-.w-blockquote::before {
-  @include font-material-icon();
-  content: 'format_quote';
-  float: left;
-  margin: 4px 0 0 -48px;
-}
-
 .w-blockquote__text {
```

Comment:
I think it might make sense to remove `.w-blockquote__text` and `.w-blockquote__cite` and replace them with:

```
blockquote > p
```

and

```
blockquote > cite
```

in the generic/_blockquote.scss file.

Reply:
done